utility/transaction_test.go

```diff
@@ -17,7 +17,7 @@ import (
 
 func TestHandleTransaction_ErrorAlreadyInMempool(t *testing.T) {
 	// Prepare test data
-	emptyTx := types.Transaction{}
+	emptyTx := coreTypes.Transaction{}
```

Comment:
I feel like you had another PR that renamed it to `core_types`, now? If so, let's stay consistent.

Reply:
yeah so i saw @bryanchriswhite mention this was a more appropriate way with the underscores. But the file already used camelCase so i left it like that just fixed the issue where it imported core types 2x with diff names. Will make them snake_case